pkgs/applications/science/math/cplex/default.nix

```diff
@@ -0,0 +1,75 @@
+{ stdenv, coreutils, requireFile, patchelf, makeWrapper, openjdk, gtk2, xorg, glibcLocales, releasePath ? null }:
```

Comment:
Can remove `patchelf`, `coreutils` and `requireFile` from here

Reply:
Done.